### ISSUE TITLE
"complexity": "off" nor does "complexity": [ "off", 20 ] disable the complexity rule

### ISSUE THREAD

**Issue by niftylettuce** [eslint/eslint#11664]
**Tell us about your environment**

* **ESLint Version:** 5.16.0
* **Node Version:** 10.15.3
* **npm Version:** 6.9.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "extends": ["eslint:recommended"],
  "plugins": ["compat"],
  "rules": {
    "complexity": "off"
  }
}
```

and/or

```js
{
  "extends": ["eslint:recommended"],
  "plugins": ["compat"],
  "rules": {
    "complexity": [ "off", 20 ]
  }
}
```

</details>

```bash
eslint file.js
```

**What did you expect to happen?**

It should have disabled the rule from being raised

**What actually happened? Please include the actual, raw output from ESLint.**

```sh
/Users/user/Projects/project/file.js
    510:119  error    Function has a complexity of 37                                                          complexity
```
